.golangci.yml

```diff
@@ -49,7 +47,6 @@ issues:
         # We should re-enable deadcode and unused linting
         # across its files, once stableswap is fully complete.
         - unused
-        - deadcode
```

Comment:
It would be great to reenable this one separately from gofumpt

Reply:
deadcode is deprecated and unused does its job.  Is that okay?

Reply:
oh, I think you mean, you'd like to enable checking unused code in stableswap?

Reply:
Yes, that's what I meant.

I saw https://github.com/osmosis-labs/osmosis/pull/5105/files. Thank you for that